apps/src/templates/rubrics/RubricSettings.jsx

```diff
@@ -246,6 +249,13 @@ export default function RubricSettings({
     tabSelectCallback(TAB_NAMES.RUBRIC);
   };
 
+  const onClickDownloadCSV = () => {
+    analyticsReporter.sendEvent(EVENTS.TA_RUBRIC_CSV_DOWNLOADED, {
+      ...(reportingData || {}),
+      sectionId: sectionId,
+    });
+  };
```

Comment:
I think I remember some discussion about this empty function in your previous PR for the CSV download. This is a good use for it!